src/collection-view.js

```diff
@@ -549,14 +561,26 @@ const CollectionView = Backbone.View.extend({
   // This will also update the indices of later views in the collection in order to keep the
   // children in sync with the collection.
   addChildView(view, index) {
+    if (!(view instanceof Backbone.View)) {
```

Comment:
I think my preference would lean towards letting it crash or throwing an error vs essentially a noop... but there might be a use-case i'm overlooking.